src/filter-dropdown.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+export default function FilterDrop(props) {
+  return (
+    <div>
+      <select defaultValue="none" className="m-3" id="topicSelect" onChange={props.sort}>
+        {props.topics
+          .map((card) => {
+            return <option key={card.cardId} value={card.topic}>{card.topic}</option>
```

Comment:
When you rework `CardList` to only pass an array of topics (Strings), each topic can be its own key, because by then, they're unique.